src/fides/lib/oauth/api/routes/user_endpoints.py

```diff
@@ -245,20 +240,19 @@ def perform_login(
             db,
             client_id_byte_length,
             client_secret_btye_length,
-            scopes=user.permissions.scopes,  # type: ignore
+            scopes=[],  # type: ignore
```

Comment:
When a user logs in, the created client doesn't have scopes, only roles or systems.  However, clients created directly can still have scopes.